snappy/dpu-compress/dpu_compress.c

```diff
@@ -35,17 +47,23 @@ static inline void advance_seqread(struct in_buffer_context *input, uint32_t len
 static inline uint32_t read_uint32(uint32_t offset)
 {
 	uint8_t data_read[16];
-	mram_read(input_buf + offset, data_read, 16);
+	mram_read(&input_buf[WINDOW_ALIGN(offset, 8)], data_read, 16);
 
 	offset %= 8;
-	
 	return (data_read[offset] |
 			(data_read[offset + 1] << 8) |
 			(data_read[offset + 2] << 16) |
-			(data_read[offset + 3] << 24));
+			(data_read[offset + 3] << 24)); 
```

Comment:
I really have nothing to say, so I'm going to pick on whitespace. It looks like the only change on this line is a space at the end of the line?

Reply:
I guess I added it by accident in my last commit...